.github/workflows/dev-docker-image.yml

```diff
@@ -0,0 +1,29 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
+    branches: [main]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: docker login
+      env:
+        DOCKER_USER: ${{ secrets.DOCKER_USER }}
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}  
+      run : |
+        docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
+    - name: Build the Docker image
+      run: |
+
+        docker-compose build efiling-demo
+        # docker tag jag-file-submission_efiling-admin:latest ${{ secrets.DOCKERHUB_ORG }}/efiling-admin:latest
+        docker tag ${{ secrets.DOCKERHUB_ORG }}/efiling-admin:latest
+        docker tag ${{ secrets.DOCKERHUB_ORG }}/efiling-admin:dev
+        docker push ${{ secrets.DOCKERHUG_ORG }}/efiling-admin:latest
```

Comment:
I'll approve, but just in case you don't know, you can do just a "docker push ${{ secrets.DOCKERHUB_ORG }}/efiling-admin" without the image tags and it'll push all tags for that host.  So in this case, one command to push them both.

You have have been explicit on purpose.

Reply:
Check the new commit.